test/src_test/parbicop_test.cpp

```diff
@@ -64,6 +67,11 @@ ParBicopTest::set_family(BicopFamily family, int rotation)
 void
 ParBicopTest::set_parameters(Eigen::VectorXd parameters)
 {
+  if (bicop_.get_family() == BicopFamily::tawn) {
```

Comment:
parameters(1) isn't used?

Reply:
In VineCopula there are only two parameters. So there are families "Tawn type I" and "Tawn type 2" where either parameters(1) or parameters(0) is fixed to 1.